File: test/level_1.c
#include <stdio.h>
#include <string.h>

int main() {
    char str[100];
    printf("Please enter a string less than 100 character, terminated by an enter key:\n");
    gets(str);

    int i = 0;
    int total = strlen(str) - 1;
    int flag = 1;

    for (i = 0; i <= total; i++) {
        if (str[i] != str[total - i]) {
            // not a palindromic string
            flag = 0;
            break;
        }
    }

    if (flag == 1) {
        printf("True\n");
    } else {
        printf("False\n");
    }

    return 0;
}

File: test/level_1.js
let printf = require("printf");
let readlineSync = require("readline-sync");
let main = () => {
    let str = new Array(100);
    while (1) {
        process.stdout.write(printf("Please enter a non-empty string less than 100 character, terminated by an enter key:\n"));
        str = readlineSync.question('').split('');
        if (str != "") {
            break;
        }
    }
    let i = 0;
    let total = str.length - 1;
    let flag = 1;
    for (i = 0;i <= total; i++) {
        if (str[i] != str[total - i]) {
            // not a palindromic string
            flag = 0;
            break;
        }
    }
    if (flag == 1) {
        process.stdout.write(printf("True\n"));
    } else {
        process.stdout.write(printf("False\n"));
    }
    return 0;
}
main();